### PR TITLE
Section footer

### PR DESCRIPTION
**Pull request by LiluUncharted**
Footer. I introduce request changes
Web capture_2-9-2023_3253_

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,44 +10,231 @@
...
+                            <a href="#"><img src="img/icon_youtube.svg" alt="Youtube"></a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+
+            <hr>
+
+            <div class="footer-container">
+                <div class="footer-columns">
```
Comment (IvanCeban) [inline index.html]: Don't forget about semantic `footer` tag

File: index.html [hunk 2]
```diff
@@ -10,44 +10,231 @@
...
+                    <div class="scan-columns-align">
```
Comment (IvanCeban) [inline index.html]: I don't really get the meaning of "scan-columns-align" class name.

**Review by IvanCeban — CHANGES_REQUESTED** [review]
Very good code structure. 
Please deal with containers and improve the class naming in the footer. The rest is great!
Good job

**Review by sdmnc — APPROVED** [review]
Html validator and CSS validator found no issues with code so

File: index.html (hunk 1 above)
Comment (LiluUncharted) [reply index.html]: I actually forgot, thanks

File: index.html (hunk 2 above)
Comment (LiluUncharted) [reply index.html]: changed names' classes

File: styles.css [hunk 3]
```diff
@@ -136,65 +150,339 @@
...
+                list-style: none;
+                margin: 0px;
+                padding: 0px;
+            }
+
+            .columns-header{
+                display: flex;
+                flex-direction: column;
+                align-items: flex-start;
+                width: 148px;
```
Comment (IvanCeban) [inline styles.css]: No fixed width please

**Review by IvanCeban — COMMENTED** [review]
Let me explain the idea with containers on the coming lesson one more time. 
I still see you didn't get my point. But you're close to. I guess I didn't explain it in a clear way.

File: styles.css (hunk 3 above)
Comment (LiluUncharted) [reply styles.css]: I tried to line everything up by design, now I have already removed the width, I hope I did it right this time

**Pull request merged**